# Linear velocity, in millimeters per second
float64 lin_vel
# Angular velocity, in millimeters per second
float64 ang_vel
# duration to apply velocities, in seconds
float64 duration
